README.md

```diff
@@ -555,15 +555,14 @@ app.launch(function(request,response) {
   response.say("Hello World");
 });
 
-// this call defines a post() and optionally a get() handler in express, mapped to the alexa-app
-// express_app: the express app instance to map to
-//        path: the path prefix to map to
-// enableDebug: when false, don't map a GET handler, default is true
-//              debugging GET requests call express' render() method using 'test'
-app.express(express_app, "/echo/", false);
-
-// now POST calls to /echo/sample in express will be handled by the app.request() function
+// ALWAYS setup the alexa app and attach it to express before anything else.
+app.express({ expressApp: express_app, router: express.Router() });
```

Comment:
`require()`ing a dependency that isn't in the application's `package.json` feels icky, even though it should work fine.

Another nice consequence of passing it in is you have more control over how you mount routes in the app. For example, in the still-open PR for we use it to pass an instance of a router instead of an express app:
https://github.com/mreinstein/alexa-app-server/blob/master/index.js#L100